ref/ams-configuration.md

```diff
@@ -26,6 +26,16 @@ AMS provides various configuration items to customise its behaviour. The followi
 | `registry.url` | string | - | URL of the application registry to use |
 | `scheduler.strategy` | string | spread | Strategy the internal container scheduler in AMS is using to distribute container across available LXD nodes. Possible values are: `binpack`, `spread` |
 
+## Node-specific configuration
+
+In a cluster setup, there are configuration items that can be customised for each node. See [Configure cluster nodes](tbd) for instructions on how to set these configuration items.
+
+| Name | Type | Default |  Description            |
+|------|------|---------|-------------------------|
+| `cpu-allocation-rate` | integer | 4 | CPU allocation rate used for [over-committing resources](tbd#overcommitting) |
+| `gpu-slots` | integer | 0 | Number of [GPU slots](tbd#gpu-slots) available on the node |
```

Comment:
Is the default 0 ?

Reply:
No, the number varies based on different GPU that is installed on that node. 

| GPU        | Number of GPU slots           | Number of GPU encoder slots  |
| ------------- |:-------------:| -----:|
| NVIDIA      | 32 | 32 |
|  AMD     |   10    |   0 |
|  Intel | 10      |    10 |